filters.go

```diff
@@ -109,6 +109,46 @@ func (p *Pipe) First(lines int) *Pipe {
 	return Echo(output.String())
 }
 
+// Last reads from the pipe, and returns a new pipe containing only the last N
+// lines. If there is an error reading the pipe, the pipe's error status is also
+// set.
+func (p *Pipe) Last(lines int) *Pipe {
+	if p == nil || p.Error() != nil {
+		return p
+	}
+	scanner := bufio.NewScanner(p.Reader)
+	var stringArr []string
```

Comment:
In Go we don't tend to name our variables after their type (as someone said, you don't name your pets "Cat" and "Dog"). Sure, this is a slice of strings (not an array), but what's it for? The name should clarify the _contents_; the compiler can already tell us the type.